bodhi/client/__init__.py

```diff
@@ -79,8 +89,20 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
-        except Exception as e:
-            traceback.print_exc()
+
+
+@updates.command()
+@add_options(common_options)
+@click.argument('builds')
+def edit(**kwargs):
```

Comment:
Even though ```new()``` doesn't have a docblock, let's give this new function one. I can give ```new()``` one sometime in the future, but it's good practice to make sure all new functions in Bodhi have docblocks.